src/main/resources/static/openapi/EducationAndWorkPlanAPI.yml

```diff
@@ -334,6 +334,33 @@ paths:
       operationId: unarchive-goal
       requestBody:
         $ref: '#/components/requestBodies/UnarchiveGoal'
+  '/action-plans/{prisonNumber}/goals/{goalReference}/complete':
+    parameters:
+      - $ref: "#/components/parameters/prisonNumberPathParameter"
+      - $ref: "#/components/parameters/goalReferencePathParameter"
+    put:
+      summary: Completes a Goal
+      description: |
+        Changes a goals status to 'COMPLETE' and records the completion note.
+
+        **Role Requirements:**
+        Access to this endpoint requires one of the following roles:
+          - `ROLE_EDUCATION_AND_WORK_PLAN__GOALS__RO` (Read-Only)
```

Comment:
Not the RO role? 🤔

Reply:
copy paste from the archive/unarchive - which are also wrong 🙄